golangci.yaml

```
@@ -0,0 +1,32 @@
+run:
+  timeout: 6m
+
+linters:
+  disable-all: true
+  enable:
+    - bodyclose
+    - deadcode
+    - dogsled
```

```suggestion
    - dogsled
    - exportloopref
```

Replaces `scopelint`:
```
WARN [runner] The linter 'scopelint' is deprecated (since v1.39.0) due to: The repository of the linter has been deprecated by the owner.  Replaced by exportloopref.
```